launchable/utils/link.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+from typing import Dict, List, Mapping
+
+JENKINS_URL_KEY = 'JENKINS_URL'
+JENKINS_BUILD_URL_KEY = 'BUILD_URL'
+GITHUB_ACTIONS_KEY = 'GITHUB_ACTIONS'
+GITHUB_ACTIONS_SERVER_URL_KEY = 'GITHUB_SERVER_URL'
+GITHUB_ACTIONS_REPOSITORY_KEY = 'GITHUB_REPOSITORY'
+GITHUB_ACTIONS_RUN_ID_KEY = 'GITHUB_RUN_ID'
+GITHUB_PULL_REQUEST_URL_KEY = "GITHUB_PULL_REQUEST_URL"
+CIRCLECI_KEY = 'CIRCLECI'
+CIRCLECI_BUILD_URL_KEY = 'CIRCLE_BUILD_URL'
+
+
+class LinkKind(Enum):
+
+    LINK_KIND_UNSPECIFIED = "LINK_KIND_UNSPECIFIED"
+    CUSTOM_LINK = "CUSTOM_LINK"
+    JENKINS = "JENKINS"
+    GITHUB_ACTIONS = "GITHUB_ACTIONS"
+    GITHUB_PULL_REQUEST = "GITHUB_PULL_REQUEST"
+    CIRCLECI = "CIRCLECI"
```

Comment:
[fyi]
In Python Enum, it is possible to string of enum key by `.name`.

```python
class LinkKind(Enum):
    LINK_KIND_UNSPECIFIED

print(LinkKind.LINK_KIND_UNSPECIFIED.name)
# LINK_KIND_UNSPECIFIED
```

Reply:
Is it true?

I tried to make sample codes then faced `NameError: name 'LINK_KIND_UNSPECIFIED' is not defined`

```python
from enum import Enum

class LinkKind(Enum):
    LINK_KIND_UNSPECIFIED

print(LinkKind.LINK_KIND_UNSPECIFIED.name)
```
Do we need to use `auto()?

Reply:
Sorry, you are right. You need auto().

```python
from enum import Enum, auto

class LinkKind(Enum):
    LINK_KIND_UNSPECIFIED = auto()

print(LinkKind.LINK_KIND_UNSPECIFIED.name)
# LINK_KIND_UNSPECIFIED
```

Reply:
How about this for Python3.5?

```python
from enum import Enum

class LinkKind(Enum):
    LINK_KIND_UNSPECIFIED = 0

print(LinkKind.LINK_KIND_UNSPECIFIED.name)
# LINK_KIND_UNSPECIFIED
```

Reply:
try to test https://github.com/launchableinc/cli/pull/529/commits/e606695dea187045a734b1b6af2149c0ba61432a